oss/sdks/ts/vector/commands/query.mdx

```diff
@@ -85,7 +91,29 @@ await index.query({ topK: 2, vector: [ ... ]})
       metadata: undefined
     },
   ],
-  namespace: ''
+}
+*/
+```
+
+```typescript Using Data
+const results = await index.query({
+  data: "Movie about an adventure of a hobbit in a fantasy world.",
+  includeVectors: true,
+  topK: 1,
+  filter: "genre = 'fantasy' and title = 'Lord of the Rings'",
```

Comment:
genre is "fanstasy" here, but "drama" below. Shall we make them the same?

Reply:
Yes, fixed